### PR TITLE
TESTING Feature/590 add skill 

### PR DESCRIPTION
**Pull request by kcinay055679**

### CODE REVIEW

File: app/controllers/skills_controller.rb [hunk 1]
```diff
@@ -1,12 +1,9 @@
 # frozen_string_literal: true
 
-require 'keycloak_tools'
-
 class SkillsController < CrudController
   include ExportController
-  include KeycloakTools
 
-  before_action :authorize_admin, except: %i[index show create unrated_by_person]
+  # before_action :authorize_admin, except: %i[index show create unrated_by_person]
```
Comment (Robin481) [inline app/controllers/skills_controller.rb:6]: delete

File: spec/features/skills_form_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    @portfolio = "passiv"
+  end
+
+  it 'creates the skill when the form is submitted' do
+    fill_in 'skill_title', with: 'New Skill Title'
+    select @category.title, from: 'skill_category_id'
+    select @radar, from: 'skill_radar'
+    select @portfolio, from: 'skill_portfolio'
+    click_button 'Skill erstellen'
+    all 'tbody tr', count: 6
```
Comment (Robin481) [inline spec/features/skills_form_spec.rb:19]: Seems like it is a little bit too unspecific. If we added a table somewhere else this test would pass for the wrong reason.
Maybe check for the 6 names of the skills instead?

File: spec/features/skills_form_spec.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+
+describe 'Skill Form', type: :system, js:true do
+  before do
+    visit skills_path
+    click_link(href: new_skill_path)
```
Comment (Robin481) [inline spec/features/skills_form_spec.rb:7]: Why do you have to set the href?

File: spec/controllers/skills_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        expect(response).to render_template("index")
+      end
+
+      it 'post returns all skills ' do
+        title = 'new skill'
+        category_id = categories(:java).id
+        radar = "hold"
+        portfolio = "passiv"
+
+        post :create , params: { skill: { title: title, category_id: category_id, radar: radar, portfolio: portfolio} }
```
Comment (Robin481) [inline spec/controllers/skills_controller_spec.rb:18]: Just pass the values straight into the creation call here instead of defining them on lines 13-16.

**Review by Robin481 — CHANGES_REQUESTED** [review]
Looking good.
Small adjustments and then merge time :rocket:

File: spec/features/skills_form_spec.rb (hunk 3 above)
Comment (kcinay055679) [reply spec/features/skills_form_spec.rb:7]: Cause it's likely we change the name of the button but the href stays consistent

File: spec/controllers/skills_controller_spec.rb (hunk 4 above)
Comment (kcinay055679) [reply spec/controllers/skills_controller_spec.rb:18]: but if i do it like this i can use the same vars to check if the value is actually the same

**Review by Robin481 — APPROVED** [review]
:rocket:

**Pull request merged**